### PR TITLE
Add sepEndBy and sepEndBy1

### PR DESCRIPTION
**Pull request by prt2121**
I happen to need them for my personal project so I sent PR here as well.

### CODE REVIEW

**Comment by Bogdanp** [comment]
Thanks for contributing. Would you mind adding some tests for these?

File: src/Combine.elm [hunk 1]
```diff
@@ -477,6 +477,25 @@ sepBy1 sep p =
...
+sepEndBy1 : Parser x -> Parser res -> Parser (List res)
+sepEndBy1 sep p =
+  p
+    `andThen` \x ->
+                (flip always)
+                  `map` sep
+                  `andMap` sepEndBy sep p
+                  `andThen` \xs ->
+                              succeed (x :: xs)
+                                `or` succeed [ x ]
```
Comment (Bogdanp) [inline src/Combine.elm]: I don't think this `or` is necessary since `succeed` will never fail

Comment (prt2121) [inline src/Combine.elm]: @Bogdanp you're right. Sorry. I'll add some tests.

**Comment by Bogdanp** [comment]
Merged. Thanks again! I've released `2.1.0` with these parsers in it.

**Comment by prt2121** [comment]
:+1:  thanks for merging. 
your library is awesome!

**Pull request merged**